index.js

```diff
@@ -11,6 +11,7 @@ function inCommand (cmd) {
   if (!npm_config_argv.cooked instanceof Array) process.exit(1)
 
   var V
+  if (npm_config_argv.cooked[0] === 'run') npm_config_argv.cooked.shift()
```

Comment:
Per #10, `lerna bootstrap` runs `npm prepublish` in each package directory to get it ready for a symlink from one of its dependencies.